pyshorteners/base.py

```diff
@@ -104,7 +104,7 @@ def _post(self, url, data=None, json=None, params=None, headers=None):
         )
         return response
 
-    def short(self, url):
+    def short(self, url ,api_key =None ):
```

Comment:
`api_key`  is provided upon Shortener object instantion, becoming an object property (`self.api_key`) . Can you remove this kwarg please?

Reply:
I have removed it as per your advice sir